docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
   ht_indexer_tracker:
     container_name: ht_indexer_tracker
     profiles: [ indexer,indexer_tests ]
-    image: ht_indexer
+    image: ghcr.io/hathitrust/indexer:unstable #ht_indexer
```

Comment:
For development, would it make sense to build the "base" image (which has all the development tools) and mount the repository, rather than using the upstream image?